codice/alliance
# CAL-163 Implement KLV security classification mapping

#### What does this PR do?

Add ability to map KLV security classification integers to security classification strings, for admins to configure the integer-to-string mapping, and for admins to control the sort order of classification strings so that the strictest classification is used when multiple classifications are present
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@bdeining 
@Lambeaux 
@kcwire 
@jlcsmith 
@stustison 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@beyelerb
@lessarderic
#### How should this be tested?
- Build and install Alliance
- configure a video stream
- use the sample-mpegts-streamgenerator to stream a video that contains security classification KLV data (ask me if you don't have a test video)
- Using the UI, check that the parent metacard and the chunk metacard contains an attribute value for "Security Classification" that says "unclassified" (assuming you used the default security mappings)
#### Any background context you want to provide?
#### What are the relevant tickets?

CAL-163
#### Screenshots (if appropriate)
#### Checklist:

…
